packages/cli/src/blueprints/package.json

```diff
@@ -79,7 +79,7 @@
     "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "test": "mt --quiet"
+    "test": "mt tests --quiet"
```

Comment:
`mt --quiet` will look for `*.test.js` files starting with the project root. By specifying the test directory (we already do this for TypeScript projects), we can limit the search and avoid potential bugs.